### PR TITLE
4.15 GA

### PR DESCRIPTION
**Pull request by kalexand-rh**
Similar to https://github.com/openshift/openshift-docs/pull/67039, the 4.14 GA PR (without the Travis changes).

@adellape, will you PTAL?

### CODE REVIEW

File: .s2i/httpd-cfg/01-commercial.conf [hunk 1]
```diff
@@ -153,7 +153,7 @@ AddType text/vtt                            vtt
 
     # Redirects for "latest" version
     RewriteRule ^(container-platform|enterprise)/?$ /container-platform/latest [R=301]
-    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.14/$2 [NE,R=301]
+    RewriteRule ^(container-platform|enterprise)/latest/?(.*)$ /container-platform/4\.15/$2 [NE,R=301]
```
Comment (kalexand-rh) [inline .s2i/httpd-cfg/01-commercial.conf:156]: @bscott-rh, will you please confirm the redirects?

File: _distro_map.yml [hunk 2]
```diff
@@ -162,7 +168,7 @@ openshift-dedicated:
     enterprise-3.11:
       name: '3'
       dir: dedicated/3
-    enterprise-4.14:
+    enterprise-4.15:
```
Comment (kalexand-rh) [inline _distro_map.yml:171]: @karanthshashank, please confirm that ROSA and OSD should start tracking 4.15 at OCP 4.15 GA.

File: _distro_map.yml [hunk 3]
```diff
@@ -240,6 +246,9 @@ openshift-telco:
     enterprise-4.14:
       name: '4.14'
       dir: container-platform-telco/4.14
+    enterprise-4.15:
```
Comment (kalexand-rh) [inline _distro_map.yml:249]: @aireilly, I'm adding the 4.15 telco distro. Please let me know if you want me to also remove the 4.14 one.

File: .s2i/httpd-cfg/01-commercial.conf (hunk 1 above)
Comment (bscott-rh) [reply .s2i/httpd-cfg/01-commercial.conf:156]: Redirects LGTM

File: _distro_map.yml (hunk 2 above)
Comment (karanthshashank) [reply _distro_map.yml:171]: ACK @kalexand-rh confirm

File: _distro_map.yml (hunk 3 above)
Comment (aireilly) [reply _distro_map.yml:249]: /lgtm

**Review by aireilly — APPROVED** [review]
/lgtm

**Pull request merged**